tools/tentacle_manager/tentacle_package_manager.py

```diff
@@ -138,11 +136,12 @@ def process_module(self, action, package, module_name, package_localisation, is_
 
                 if module_test_files:
                     for test in module_tests:
-                        test_loc = "{0}.py".format(TentacleUtil.create_localization_from_type(package_localisation,
-                                                                                              module_type,
-                                                                                              module_subtype,
-                                                                                              test,
-                                                                                              True))
+                        location_str = TentacleUtil.create_localization_from_type(package_localisation,
```

Comment:
Why did you need to split this line up ?

Reply:
readability

Reply:
Ok why not